### PR TITLE
Export verified variants from all institutes when user is admin

### LINKED ISSUE #3109: Download verified variants is locked to home institute
Dashboard downloads should allow admin override, since admin can anyway search and view both the individual cases and summary statistics. As it is now, the download button is shown regardless of the search the admin performed, but the download will execute for the home institute. Thanks to @jemten for noticing! 😸

### PR DESCRIPTION
**Pull request by northwestwitch**
Fix #3109

<details>
<summary>Testing on cg-vm1 server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. Make sure the PR is pushed and available on [Docker Hub](https://hub.docker.com/repository/docker/clinicalgenomics/scout-server-stage)
1. `ssh your.email@cg-vm1.scilifelab.se`
1. `sudo -iu hiseq.clinical`
1. `ssh localhost`
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `podman ps`
1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
1. Make sure the branch is deployed: `systemctl --user status scout.target`
</details>

**How to test**:
1. Install master branch in cg-vm1 
1. As an admin, go to the dashboard and run an empty query (all institutes)
1. You should see that you downloaded some files, one for each of your institutes
1. Switch to this branch and repeat

**Expected outcome**:
- [ ] You should be able to download validates variants from more institutes now 

**Review:**
- [x] code approved by DN
- [x] tests executed by CR

### CODE REVIEW

**Comment by northwestwitch** [comment]
Downloaded files using master branch:

image

Downloaded files using THIS branch:

image

File: scout/server/blueprints/variants/views.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
 import shutil
 
 import pymongo
-from flask import Blueprint, abort, current_app, flash, redirect, request, send_file, url_for
+from flask import Blueprint, flash, redirect, request, send_file, url_for
```
Comment (dnil) [inline scout/server/blueprints/variants/views.py:9]: 🧹

File: scout/server/blueprints/variants/views.py [hunk 2]
```diff
@@ -533,8 +533,12 @@ def upload_panel(institute_id, case_name):
 @variants_bp.route("/verified", methods=["GET"])
 def download_verified():
     """Download all verified variants for user's cases"""
-    user_obj = store.user(current_user.email)
-    user_institutes = user_obj.get("institutes")
+
+    user_institutes = (
```
Comment (dnil) [inline scout/server/blueprints/variants/views.py:537]: Simple and effective! 🥇

File: scout/server/blueprints/variants/views.py [hunk 3]
```diff
@@ -533,8 +533,12 @@ def upload_panel(institute_id, case_name):
...
+        [inst["_id"] for inst in store.institutes()]
+        if current_user.is_admin
+        else current_user.get("institutes")
```
Comment (dnil) [inline scout/server/blueprints/variants/views.py:540]: Oh, but that won't work for non admins, right? LoginUser is a UserMixin, not a Scout user obj/dict?
```
2022-01-27 13:04:58 k1315C02VR0WRHTDH scout.server.app[88833] ERROR Exception on /verified [GET]
Traceback (most recent call last):
  File "/usr/local/miniconda3/envs/scout39/lib/python3.9/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/miniconda3/envs/scout39/lib/python3.9/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/miniconda3/envs/scout39/lib/python3.9/site-packages/flask_cors/extension.py", line 165, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/miniconda3/envs/scout39/lib/python3.9/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/miniconda3/envs/scout39/lib/python3.9/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/usr/local/miniconda3/envs/scout39/lib/python3.9/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/miniconda3/envs/scout39/lib/python3.9/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/Users/dannil/sandbox/scout/scout/server/blueprints/variants/views.py", line 540, in download_verified
    else current_user.get("institutes")
  File "/usr/local/miniconda3/envs/scout39/lib/python3.9/site-packages/werkzeug/local.py", line 347, in __getattr__
    return getattr(self._get_current_object(), name)
AttributeError: 'LoginUser' object has no attribute 'get'
```

**Review by dnil — CHANGES_REQUESTED** [review]
Suggest reverting to using user_obj to fetch institutes for non privileged users!

File: scout/server/blueprints/variants/views.py (hunk 3 above)
Comment (dnil) [reply scout/server/blueprints/variants/views.py:540]:
```suggestion
        else user_obj.get("institutes")
```

File: scout/server/blueprints/variants/views.py (hunk 2 above)
Comment (dnil) [reply scout/server/blueprints/variants/views.py:537]:
```suggestion
    user_obj = store.user(current_user.email)

    user_institutes = (
```

**Comment by northwestwitch** [comment]
> Suggest reverting to using user_obj to fetch institutes for non privileged users!

Right, otherwise it crashes, good suggestion! :)

**Pull request merged**